django/db/models/lookups.py

```diff
@@ -151,7 +151,7 @@ def process_lhs(self, compiler, connection, lhs=None):
         lhs_sql = connection.ops.field_cast_sql(
             db_type, field_internal_type) % lhs_sql
         lhs_sql = connection.ops.lookup_cast(self.lookup_name, field_internal_type) % lhs_sql
-        return lhs_sql, params
+        return lhs_sql, list(params)
```

Comment:
I think params[:] would be more fitting here. This is more explicit in saying "return a copy", and list(params) could turn something that isn't a list into a list, thus making debugging harder.